### PR TITLE
NO-TICKET :: NCTL :: Users are delegators

### PR DESCRIPTION
**Pull request by ghost**
1. Each test user account is now a delegator by default.

### CODE REVIEW

File: utils/nctl/activate [hunk 1]
```diff
@@ -135,7 +135,7 @@ alias nctl-kv-storage-install='source $NCTL/sh/contracts-kv/do_install.sh'
 alias nctl-kv-storage-set-key='source $NCTL/sh/contracts-kv/set_key.sh'
 
 # Contracts #5: Counter.
-# alias nctl-counter-install='source $NCTL/sh/contracts-counter/do_install.sh'
-# alias nctl-counter-increment='source $NCTL/sh/contracts-counter/do_increment.sh'
+alias nctl-counter-install='source $NCTL/sh/contracts-counter/do_install.sh'
+alias nctl-counter-increment='source $NCTL/sh/contracts-counter/increment.sh'
 # alias nctl-counter-decrement='source $NCTL/sh/contracts-counter/do_decrement.sh'
-# alias nctl-counter-view='source $NCTL/sh/contracts-counter/view.sh'
+alias nctl-counter-view='source $NCTL/sh/contracts-counter/view.sh'
```
Comment (goral09) [inline utils/nctl/activate:141]: Unrelated changes?

**Review by goral09 — COMMENTED** [review]
Can you remove the changes related to the counter contract from this PR? I don't feel knowledgeable enough to approve those.

**Comment by momipsl** [comment]
> Can you remove the changes related to the counter contract from this PR? I don't feel knowledgeable enough to approve those.

@goral09 Done

**Review by goral09 — APPROVED** [review]
